test/LibraryManager.IntegrationTest/LibmanTestsUtility.cs

```diff
@@ -8,6 +9,142 @@ namespace Microsoft.Web.LibraryManager.IntegrationTest
 {
     public class LibmanTestsUtility
```

Comment:
This utility class is just becoming a dumping ground for various helpers.  Can you organize these to make them discoverable to new test authors in the future?  This has been a significant compaint about test code on our team for a while, let's not repeat past mistakes.

Reply:
Seperate it into two helper class

---
In reply to: [195513583](https://github.com/aspnet/LibraryManager/pull/113#discussion_r195513583) [](ancestors = 195513583)